Homeworks/HW_#3_Class/app3.js

```diff
@@ -0,0 +1,55 @@
+class Vector {
+    constructor(vectorArray) {
+        this.vectorArray = vectorArray;
+    }
+
+    #getValidVector(inputtedVector) {
+        if (inputtedVector.vectorArray.length === this.vectorArray.length) {
+            return true
+        }
+        throw "Error"
+    }
+
+    add(inputtedVector) {
+        this.#getValidVector(inputtedVector)
+        return this.vectorArray.map(function (num, index) {
```

Comment:
Please try to write more consistent code. Here you pass to the map method an anonymous function declared with the function keyword, but In a similar situation, you pass the anonymous arrow function to the reduce method. Please choose one approach and stick to it, that'll make your code look cleaner.

In general, in such cases mostly anonymous arrow functions are used, since they have more concise syntax.
For example
`[1, 2, 3].map(el => el + 2)`